kopeme-core/src/main/java/de/dagere/kopeme/AndroidConfiguration.java

```diff
@@ -0,0 +1,25 @@
+package de.dagere.kopeme;
+import java.io.InputStream;
+import de.dagere.kopeme.junit.rule.annotations.KoPeMeConstants;
+
+/**
+ * Configuration class for Android projects to read environment variables, properties, etc
+ * from a config.json file in src/main/resources.
+ */
+public class AndroidConfiguration {
+    
+    //  Configuration file name for Android projects inside main/src/resources folder.
+    static final String ANDROID_CONFIG = "kopeme_config.json";
+
+    public static String read(String fieldName) {
+        InputStream inputStream = AndroidConfiguration.class.getClassLoader().getResourceAsStream(ANDROID_CONFIG);
+        String fieldValue;
+        try {
+            fieldValue = KoPeMeConstants.OBJECTMAPPER.readTree(inputStream).get(fieldName).asText();
```

Comment:
Could you split this line and declare local variables for readability?

Reply:
Hopefully this is more readable https://github.com/DaGeRe/KoPeMe/pull/61/commits/d9348f06fc6f84883e073a77f5b15eb4a1ae722c